ballerina/tests/test.bal

```diff
@@ -84,6 +84,46 @@ function testCreateObject() {
     }
 }
 
+@test:Config {
+    dependsOn: [testGetObject]
+}
+function testCreatePresignedURLGET() returns error? {
+    log:printInfo("amazonS3Client->createPresignedURL() GET");
+    Client|error amazonS3Client = new (amazonS3Config);
+    if amazonS3Client is Client {
+        string|error? response = amazonS3Client->createPresignedURL(testBucketName, fileName, GET, 3600);
+        if response is string {
+            http:Client cl = check new (response);
+            http:Response a = check cl->get("");
+            test:assertEquals(a.statusCode, 200, "Failed to create presigned URL");
+        } else if response is error {
+            test:assertFail("Failed to create presigned URL");
+        }
+    } else {
+        test:assertFail(amazonS3Client.toString());
+    }
+}
+
+@test:Config {
+    dependsOn: [testGetObject]
+}
+function testCreatePresignedURLPUT() returns error? {
+    log:printInfo("amazonS3Client->createPresignedURL() PUT");
+    Client|error amazonS3Client = new (amazonS3Config);
+    if amazonS3Client is Client {
+        string|error? response = amazonS3Client->createPresignedURL(testBucketName, fileName, PUT, 3600);
+        if response is string {
+            http:Client cl = check new (response);
+            http:Response a = check cl->put("", content);
+            test:assertEquals(a.statusCode, 200, "Failed to create presigned URL");
+        } else {
+            test:assertFail("Failed to create presigned URL");
+        }
+    } else {
+        test:assertFail(amazonS3Client.toString());
+    }
```

Comment:
we replace the else blocks (for error checking) with check statements to simplify the code. Please update all the test cases accordingly.

```suggestion
    Client amazonS3Client = check new (amazonS3Config);
    string response = amazonS3Client->check createPresignedURL(testBucketName, fileName, PUT, 3600);
    
    http:Client httpClient = check new (response);
    http:Response response2 = check httpClient->put("", content);
    test:assertEquals(response2.statusCode, 200, "Failed to create presigned URL");
```

Reply:
Also please follow the general naming convention for Ballerina variable names in here.

Reply:
> Please update all the test cases accordingly

All test cases meaning the tests for the already available functions (testCreateBucket ,testListBuckets... etc)?

Reply:
> All test cases meaning the tests for the already available functions (testCreateBucket ,testListBuckets... etc)?

Whats I meant was all the newly added test cases :)